<launch>
<!-- Launch file for INNO LiDAR -->

	<arg name="rviz" default="true" />
 	<!-- <node pkg="rosbag" type="play" name="rosbag_play" args="-clock -r 0.5 /home/xng/data/Test/odom_test_dataset/3_tunnel/rosbag/tunnel_4.bag"/>  -->
	<node pkg="rosbag" type="play" name="rosbag_play" args="--clock -r 0.5 /home/xng/data/Test/odom_test_dataset/4_wuyue/rosbag/wuyue_001.bag /home/xng/data/Test/odom_test_dataset/4_wuyue/rosbag/wuyue_002.bag"/>

	<rosparam command="load" file="$(find fast_lio)/config/inno.yaml" />

	<param name="feature_extract_enable" type="bool" value="1"/>
	<param name="point_filter_num" type="int" value="3"/>
	<param name="max_iteration" type="int" value="3" />
	<param name="filter_size_surf" type="double" value="0.5" />
	<param name="filter_size_map" type="double" value="0.5" />
	<param name="cube_side_length" type="double" value="1000" />
	<param name="runtime_pos_log_enable" type="bool" value="0" />
<!--	<param name="use_sim_time" value="false"/> -->
    <node pkg="fast_lio" type="fastlio_mapping" name="laserMapping" output="screen" /> 

	<group if="$(arg rviz)">
	<node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find fast_lio)/rviz_cfg/loam_livox.rviz" />
	</group>

</launch>
